external-crates/move/crates/move-vm-runtime/src/execution/vm.rs

```diff
@@ -137,6 +142,47 @@ impl<'extensions> MoveVM<'extensions> {
             function_name,
             ty_args,
             args,
+            None,
+            gas_meter,
+            bypass_declared_entry_check,
+        )
+    }
+
+    /// Similar to execute_entry_function, but it bypasses visibility checks and accepts a tracer
+    pub fn execute_function_bypass_visibility_with_tracer_if_enabled(
```

Comment:
Not for this PR, but thoughts on eventually rolling this into one function `execute_function_bypass_visibility`?